cloud_functions/ads_policy_monitor/init.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+bucketname='ads_policy_monitor'
```

Comment:
This needs to be globally unique, so it might be worth concatenating this with the project name (or asking for user input).

Reply:
I changed to fetch the bucket name from vars file + added script exit if bucket creation doesn't succeed